### PR TITLE
modify transactions

### PR DESCRIPTION
**Pull request by ToniyaSundaram**
- refactored send, bond and unbond transactions
-  updated config files with permissions

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -117,28 +117,28 @@ var actions = new Actions(config);
...
-  .action((priv_key,address,amount,fee,public_key,cmd) => actions.broadcastBond(priv_key,address,parseInt(amount),parseInt(fee),public_key,cmd.unsafe));
+  .action((address,amount,fee,public_key,priv_key) => actions.broadcastBond(address,parseInt(amount),parseInt(fee),public_key,priv_key));
 
   program
-  .command('unbond <priv_key> <address> <amount> <fee>')
+  .command('unbond <address> <amount> <fee> <priv_key>')
   .option('-u, --unsafe', 'unsafe sending transaction')
   .alias('ubnd')
-  .description('\n(safe) Do Unbond transaction, you need pass the private key of sender and address of receiver\
+  .description('\n(safe) Do Unbond transaction, you need pass address of the sender \
```
Comment (b00f) [inline index.js]: To make it more clear we can state it here that sender is a validator indeed

File: index.js [hunk 2]
```diff
@@ -117,28 +117,28 @@ var actions = new Actions(config);
...
   \nyou may need to initialize a project before using this command.\n\n')
-  .action((priv_key,address,amount,fee,cmd) => actions.broadcastUnbond(priv_key,address,parseInt(amount),parseInt(fee),cmd.unsafe));
+  .action((address,amount,fee,priv_key) => actions.broadcastUnbond(address,parseInt(amount),parseInt(fee),priv_key));
 
   program
-  .command('send <priv_key> <address> <amount> ')
-  .option('-u, --unsafe', 'unsafe sending transaction')
+  .command('send <toAddress> <amount> <priv_key>')
```
Comment (b00f) [inline index.js]: To make this command more compatible with others I suggest that we use `address` instead of `toAddrres` or vice versa.

**Comment by b00f** [comment]
@Kishanjvaghela Thanks.

File: index.js [hunk 3]
```diff
@@ -117,28 +117,28 @@ var actions = new Actions(config);
   .action((priv_key,data,address,fee,gas_limit,cmd) => actions.transact(priv_key,data,address,fee,gas_limit,cmd.unsafe));
 
   program
-  .command('bond <priv_key> <address> <amount> <fee> <public_key>')
-  .option('-u, --unsafe', 'unsafe sending transaction')
+  .command('bond <address> <amount> <fee> <public_key> <priv_key>')
```
[the remark this answers is not in the document]
Comment (Kishanjvaghela) [reply index.js]: Later we planned to make it with pass phrase. For that we need address.

[the remark this answers is not in the document]
Comment (b00f) [reply index.js]: @Kishanjvaghela You can extract address from the public key. It looks having both is redundant here.

File: index.js (hunk 1 above)
Comment (ToniyaSundaram) [reply index.js]: sure noted

File: index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ToniyaSundaram) [reply index.js]: Yes Mostafa . we can make it even more simpler

[the remark this answers is not in the document]
Comment (Kishanjvaghela) [reply index.js]: ok👍🏻

File: index.js (hunk 1 above)
Comment (ToniyaSundaram) [reply index.js]: sure noted

File: index.js (hunk 2 above)
Comment (ToniyaSundaram) [reply index.js]: Sure I will implement it

File: index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ToniyaSundaram) [reply index.js]: @b00f  implemented in commit `074177471916e32518ee87b928ce7043919c6287`

File: index.js (hunk 2 above)
Comment (ToniyaSundaram) [reply index.js]: modified in commit `ec75375076fba8677df1b3f351223463f5e392e9`

File: index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (b00f) [reply index.js]: @ToniyaSundaram thanks

**Pull request merged**